src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/ResourceFormStore.js

```diff
@@ -87,15 +87,114 @@ export default class ResourceFormStore extends AbstractFormStore implements Form
         });
     };
 
-    @action handleSchemaResponse = ([schema, jsonSchema]: [Schema, Object]) => {
+    handleSchemaResponse = ([schema, jsonSchema]: [Schema, Object]) => {
         this.validator = jsonSchema ? ajv.compile(jsonSchema) : undefined;
         this.pathsByTag = {};
 
-        this.schema = schema;
-        this.addMissingSchemaProperties();
-        this.setSchemaLoading(false);
+        return this.requestRemoteDataAndMerge(this.schema, schema).then(action(() => {
+            this.schema = schema;
+            this.addMissingSchemaProperties();
+            this.validate();
+            this.setSchemaLoading(false);
+        }));
     };
 
+    requestRemoteDataAndMerge = (localSchema: Schema, remoteSchema: Schema) => {
+        if (localSchema) {
```

Comment:
```suggestion
        // load data only after initial schema was set to prevent duplicate requests during initialization
        if (localSchema) {
```